process_report/process_report.py

```diff
@@ -302,6 +310,55 @@ def export_HU_only(dataframe, output_file):
     HU_projects.to_csv(output_file)
 
 
+def export_BU_only(dataframe: pandas.DataFrame, output_file):
+    def get_project(row):
+        project_alloc = row[PROJECT_FIELD]
+        if project_alloc.rfind("-") == -1:
+            return project_alloc
+        else:
+            return project_alloc[: project_alloc.rfind("-")]
+
+    BU_projects = dataframe[dataframe[INSTITUTION_FIELD] == "Boston University"]
+    BU_projects["Project"] = BU_projects.apply(get_project, axis=1)
+    BU_projects[SUBSIDY_FIELD] = 0
+    BU_projects = BU_projects[
+        [
+            INVOICE_DATE_FIELD,
+            PI_FIELD,
+            "Project",
+            COST_FIELD,
+            CREDIT_FIELD,
+            SUBSIDY_FIELD,
+            BALANCE_FIELD,
+        ]
+    ]
+    BU_projects = _apply_BU_subsidy(BU_projects)
```

Comment:
Running this here (after already having exported the billable file) is going to cause the balance fields to be different in the billables invoice and the BU only invoice. @joachimweyl is this something we care about?